test/icon-lookup.c

```diff
@@ -55,20 +46,6 @@ TEST test_find_icon(void)
         PASS();
 }
 
-TEST test_find_path(void)
```

Comment:
Why did you remove these tests? Did anything in the search behavior change other than the default icons? If so that should also be documented and discussed

Reply:
find_icon_path and get_path_from_icon had basically half of the code duplicate. so I removed some code from the first since in the first place it is used only by get_path_from_icon

Reply:
the absolute path and uri handling were the exact same

Reply:
Ah okay. Makes sense